airflow/jobs/scheduler_job.py

```diff
@@ -751,7 +751,7 @@ def _change_state_for_tis_without_dagrun(self,
         changed manually.
 
         :param old_states: examine TaskInstances in this state
-        :type old_state: list[airflow.utils.state.State]
+        :type old_states: list[airflow.utils.state.State]
```

Comment:
Hi @kaxil , seems there is the same error in line 758. Do you mind fixing it together?

Reply:
Done :)